CHANGELOG.rst

```diff
@@ -10,6 +10,7 @@ Added
 =====
 - Support "untagged" and "any" on EVCs.
 - `PUT /trace and /traces` endpoints validate payload with ``@validate_openapi`` from kytos.core.
+- Add ``v1`` on API routes. 
```

Comment:
@gretelliz could you please move it to the `Changed` section? Although without a v1 it wasn't fully considered "official stable" yet, but it was indeed a breaking change from what was previously released.